adam/perception/perception_frame_difference.py

```diff
@@ -0,0 +1,72 @@
+from abc import ABC
+from typing import Generic, TypeVar
+
+from attr import attrib, attrs
+from attr.validators import in_, instance_of
+from immutablecollections import ImmutableSet
+from adam.perception.developmental_primitive_perception import DevelopmentalPrimitivePerceptionFrame, ObjectPerception, \
+    PropertyPerception, RelationPerception
+
+
+class PerceptionFrameDiff(ABC):
```

Comment:
I think we can delete this class for now and add it in the future if we decide it will provide value then.